concepts/Vulnerability_disclosure_and_embargo_policy/00004-Vulnerability_disclosure_and_embargo_policy.md

```diff
@@ -0,0 +1,251 @@
+Security Advisories."
+
+--- 
+### Vulnerability Reporting and Embargo Policy
+The Enarx project aims to be as safe to use as possible and therefore to 
+produce code that is as secure as possible. As such, we welcome any and all 
+research into Enarx and appreciate your efforts to take the time to 
+responsibly disclose any issues you find.
+
+Vulnerabilities may be discovered in:
+1. Enarx code
+2. Enarx dependencies (including compilers)
+3. Enarx documentation
+4. 3rd party hardware or firmware
+
+All security bugs in Enarx should be reported to the security team. 
+To do so, please reach out either by email at security@enarx.io or on our
+[chat](https://chat.enarx.io) stating you'd like to report a security
+issue, but **without divulging details**.
+
+We treat both email and chat as a public channels, so simply saying that
+you’re trying to get a hold of someone from the security team is fine.
+
+From then, we will establish a more secure area where the issue can be
+discussed, in the form of a 
+[Github Security Advisory](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories).
+
+You will be invited to join this private area to discuss specifics. Doing so 
+allows us to start with a high level of confidentiality and relax it if the 
+issue is less critical, moving to work on the fix in the open.
+
+This methods provides several other advantages :
+1. It avoids email, which is unfortunately very hard dificult to use securely
+2. It makes it possible to privately collaborate to fix the vulnerability in 
+a temporary private fork
+3. It makes it possible to draft a security advisory, and use the draft to 
+privately discuss the impact of the vulnerability on the project.
+4. It makes it possible to easily publish the security advisory and alert the 
+community of the vulnerability
+
+Your initial contact will be acknowledged within 48 hours, and you’ll receive 
+a more detailed response within 96 hours indicating the next steps in handling 
+your report.
+
+After the initial reply to your report, the security team  will endeavor to 
+keep you informed of the progress being made towards a fix and full 
+announcement. As recommended by 
+[RFPolicy](https://dl.packetstormsecurity.net/papers/general/rfpolicy-2.0.txt),
+ these updates will be sent at least every five working days.
+
+If you have not received a reply to your initial contact within 72 hours, or
+have not heard from the security team for the past five days, there are a few
+steps you can take (in suggested order):
+- Contact the current security coordinator (Nathaniel McCallum, aka
+[npmacallum](https://github.com/npmccallum)) directly
+- Contact the back-up contact (Mike Bursell, aka 
+[MikeCamel](https://github.com/mikecamel)) directly  
+  In both these cases, you can reach out
+  - by email
+  - by chat
+- Come talk to us on the [chat](https://chat.enarx.io).
```

Comment:
I thought it was a little ambiguous who to reach out to on the chat should the reporter bypass Nathaniel and Mike, though realistically if they reached out via the chat in the first place the team would see the disclosure information. 
```suggestion
[npmccallum](https://github.com/npmccallum)) directly
- Contact the back-up contact (Mike Bursell, aka 
[MikeCamel](https://github.com/mikecamel)) directly  
  In both these cases, you can reach out
  - by email
  - by chat
- Contact the team as a whole on the [chat](https://chat.enarx.io).
```

Reply:
> though realistically if they reached out via the chat in the first place the team would see the disclosure information.

The intention is to encourage initial disclosure of the fact that there _is_ a vulnerability, and then move to confidential channel to disclose what it is.